alot/commands/globals.py

```diff
@@ -770,7 +771,7 @@ def apply(self, ui):
                           priority='error')
                 return
             try:
-                with open(path) as f:
+                with codecs.open(path, encoding='utf-8') as f:
```

Comment:
We use magic for making educated guesses about encoding, and we have a wrapper in  `alot.helpers.guess_encoding`.

something like this probably:
```python
with open(path, 'rb') as f:
     blob = f.read()
encoding = helpers.guess_encoding(blob)
self.envelope.parse_template(blob.decode(encoding))
```